cmd/build.go

```diff
@@ -169,7 +169,3 @@ An optional flag --tag/-t is now available to override the resulting docker imag
 
 	return buildCmd
 }
-
-func init() {
-	rootCmd.AddCommand(BuildCmd())
-}
```

Comment:
I feel init functions are generally an anti-pattern and should be avoided. 

Although it can be useful for unique purposes, it is generally not recommended to use package-level init() functions because they can have unintended side effects and make it difficult to reason about the behavior of the code.

Particularly:
- Order of execution is [not obvious](https://medium.com/@markbates/go-init-order-dafa89fcef22).
- Global state is generally an anti-pattern. 

* https://github.com/leighmcculloch/gochecknoinits
* https://liyafu.com/2021-07-07-the-problems-of-go-init-func/
* https://github.com/golang/go/issues/43731